### ISSUE TITLE
A11y issues due to gallery not responding to high contrast theme changes

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#232]
Found during accessibility grading.

Tracking internal bug: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40136353
"Note: Same issue is observed in TrackPlayer -> KB focus is not visible on play/pause/… buttons."

**Comment by AgneLukoseviciute** [comment]
reproed, the whole page appears to be a mix of light mode, dark mode, and high contrast which I think might be the issue -- the focus border for the button is white but because the surrounding view is white/light, it's not visible. I think the work here is to actually have the whole app cohesively respond to theme changes.
